### PR TITLE
[luci] Import use shape_status

### PR DESCRIPTION
**Pull request by seanshpark**
This will revise import to use shape_status and replace no_shape

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #1610, from draft #1363

**Comment by seanshpark** [comment]
@mhs4670go @llFreetimell @jinevening PTAL

File: compiler/luci/import/src/GraphBuilder.cpp [hunk 1]
```diff
@@ -48,19 +50,18 @@ void GraphBuilder::build(const circle::OperatorT &op, GraphBuilderContext *conte
   {
     const circle::TensorT &output_tensor = *tensors[outputs[0]];
     copy_tensor_attributes(output_tensor, node);
+    // mark shape_status
+    if (tensors_ptr->Get(outputs[0]) == nullptr)
```
Comment (mhs4670go) [inline compiler/luci/import/src/GraphBuilder.cpp:54]: May I have a question? Is it impossible to get info like `tensors_ptr->Get(outputs[0]` from above `tensors`? like `tensors.at(outputs[0])`.

Comment (seanshpark) [reply compiler/luci/import/src/GraphBuilder.cpp:54]: Um.. existing code is incorrect. We need to check shape existance.
That is shoud be `if (tensors_ptr->Get(outputs[0])->shape() == nullptr)`

Comment (seanshpark) [reply compiler/luci/import/src/GraphBuilder.cpp:54]: Thank you for giving me a second chance to look at the code.

**Comment by seanshpark** [comment]
@mhs4670go , PTAL again

**Pull request merged**